File: src/main/java/de/hinundhergestellt/jhuh/vendors/ready2order/model/ProductsPostRequestMixin.java
package de.hinundhergestellt.jhuh.vendors.ready2order.model;

import com.fasterxml.jackson.annotation.JsonIgnore;

@SuppressWarnings("unused")
public abstract class ProductsPostRequestMixin {

    @JsonIgnore
    public abstract Integer getProductVatId();

    @JsonIgnore
    public abstract Integer getProductgroupId();
}

File: hinundhergestellt-server/src/main/kotlin/de/hinundhergestellt/jhuh/vendors/ready2order/openapi/model/ProductsPostRequestMixin.kt
package de.hinundhergestellt.jhuh.vendors.ready2order.openapi.model

import com.fasterxml.jackson.annotation.JsonIgnore

@Suppress("unused")
abstract class ProductsPostRequestMixin {

    @get:JsonIgnore
    abstract val productVatId: Int?

    @get:JsonIgnore
    abstract val productgroupId: Int?
}
